kobe_rick_and_morty_challenge/pubspec.yaml

```
@@ -0,0 +1,95 @@
+    sdk: flutter
+
+  dio: ^5.4.3+1
+  google_fonts: ^6.3.0
+  cupertino_icons: ^1.0.8
+
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  mockito: ^5.4.4
+  build_runner: ^2.4.9 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)